<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
<<<<<<< HEAD
    <width>940</width>
    <height>661</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="numero1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>300</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
=======
    <width>428</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>80</width>
      <height>21</height>
     </rect>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="numero3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>3</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>4</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero5">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>5</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero6_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>360</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>6</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero7_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>420</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>7</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>420</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>8</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>420</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>9</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QPushButton" name="numero0">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>480</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
=======
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>290</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
    <property name="text">
     <string>0</string>
    </property>
   </widget>
<<<<<<< HEAD
   <widget class="QLineEdit" name="hankiID">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>43</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="nappiok">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>480</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nappikorjaa">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>480</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="otsikko1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>491</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Syötä pankkikorttisi ID ja paina OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="hankiPIN">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
=======
  </widget>
>>>>>>> b8222f1ae90ea875b6a1d7f71e0bfa1253c71f6f
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
